packages/eslint-plugin/docs/rules/no-duplicate-type-union-intersection-members.md

```diff
@@ -0,0 +1,49 @@
+---
+description: 'Disallow duplicate union/intersection type members.'
+---
+
+> 🛑 This file is source code, not the primary documentation location! 🛑
+>
+> See **https://typescript-eslint.io/rules/no-duplicate-type-union-intersection-members** for documentation.
```

Comment:
"Members" typically refers to class/object members. The technical term here is _constituents_, I think. To match with https://typescript-eslint.io/rules/no-redundant-type-constituents, how about...

```suggestion
> See **https://typescript-eslint.io/rules/no-duplicate-type-constituents** for documentation.
```

Aside: I had a jolt of panic looking at this issue because at first I misinterpreted it to be a dup of that rule. But these are different, phew!

Reply:
> "Members" typically refers to class/object members. The technical term here is constituents, I think. 

Thanks for the useful info! (I'm not yet familiar with the nuances of English technical terms.... So it was very helpful 👍 ).
I changed the rule names and file names as suggested.